lib/new_relic/agent/instrumentation/active_record_helper.rb

```diff
@@ -153,20 +165,16 @@ def map_operation(raw_operation)
         PRODUCT_NAMES = {
           "mysql" => "MySQL",
           "mysql2" => "MySQL",
-          "mysql2_makara" => "MySQL",
```

Comment:
Use the new chokepoint method, `bare_adapter_name` instead of having every possible suffix based adapter name defined in this hash.